cocos/core/algorithm/binary-search.ts

```diff
@@ -65,13 +65,13 @@ export function binarySearchEpsilon (array: Readonly<ArrayLike<number>>, value:
 }
 
 /**
- * Searches the **sorted** array for an element and returns the index of that element.
+ * Searches the **ascending sorted** array for an element and returns the index of that element.
  * @param array The array to search in.
  * @param value The value to search.
- * @param lessThan The predicate which implements the less than semantic.
- * @return The index of the searched element in the sorted array, if one is found;
- * otherwise, a negative number that is the bitwise complement of the index of the next element that is large than the searched value or,
- * if there is no larger element(include the case that the array is empty), the bitwise complement of array's length.
+ * @param lessThan Comparison function object which returns ​true if the first argument is less than the second.
```

Comment:
`Comparison function` -> `The comparison function`

Reply:
I referred to STL documentation: https://en.cppreference.com/w/cpp/algorithm/sort. It doesn't use `the`:
![WeChatWorkScreenshot_a716f556-d04b-47a8-9bae-a6198197753c](https://user-images.githubusercontent.com/488203/208843961-689e4519-1b0d-4554-a1ea-523accca8ae4.png)